homework-submission/myapp/src/App.js

```diff
@@ -0,0 +1,24 @@
+import React, { Component } from "react";
+import "./App.css";
+import logo from "./logo.svg";
+import ItemList from "./components/itemList";
+import AddMenu from "./components/addMenu";
+import { observer, inject } from "mobx-react";
+
+@inject("todoStore")
+@observer
```

Comment:
I would prefer to not inject in a component if no need to extract the store from it in case of use.